README.md

```diff
@@ -5,52 +5,68 @@
 
 ## Quickstart
 
-Before you can begin with Yari, you need [Content](https://github.com/mdn/content).
-See its README which basically, says something like this:
+Before you can start working with Yari, you need a copy of the MDN [Content](https://github.com/mdn/content)
```

Comment:
Is it worth saying "who/when should be doing this"? The only time I had to clone yari was to test a yari defect.

Reply:
We had a paragraph that explains this a lot further down. I've edited it and moved it up:

Development on `yari` involves updating the machinery that renders MDN content
or improving the structure and styling of the MDN UI (e.g. the
styling of the header). If you are more interested in contributing to the MDN
content, you should check out the [content](https://github.com/mdn/content) repo
README instead.